src/twin/mod_test.rs

```diff
@@ -258,32 +258,32 @@ pub mod mod_test {
                     "module-version": env!("CARGO_PKG_VERSION"),
                     "azure-sdk-version": IotHubClient::sdk_version_string()
                 })))
-                .times(1)
+                .times(2)
```

Comment:
why twice? seems i don't understand the change then.
can you point me to the triggers?

Reply:
In lines 375+376 I connect twin twice in order to test if we report all properties again on a 2nd call